nuxt.config.js

```diff
@@ -83,7 +83,6 @@ export default {
             ...script
         ]
     },
-    loadingIndicator: '~/components/Loading/index.html',
```

Comment:
Did you get to dislike that loading indicator?

Reply:
I put reasoning in the commit description

![Screenshot 2020-05-30 at 01 49 07](https://user-images.githubusercontent.com/8293444/83315421-c0c05880-a217-11ea-99fd-2a7058ad65e0.png)

Reply:
This is the page that's shown before the whole app loads, so it should be kept.